frontend/src/pages/Author.jsx

```diff
@@ -37,6 +46,8 @@ class Author extends Component {
             .then((httpResponse) => {
                 if (httpResponse.status === 200) {
                     httpResponse.json().then((results) => {
+                        var authorID;
```

Comment:
You can do this in one line.
const authorID = this.getloggedinAuthorIDandHost()

Reply:
ok